.rubocop.yml

```diff
@@ -0,0 +1,70 @@
+require: rubocop-rspec
```

Comment:
Not sure that you need all this configuration:
- Lint/AmbiguousBlockAssociation
- Metrics/BlockLength
- Metrics/ModuleLength
- Rails/UnknownEnv
- Style/SymbolArray
- RSpec/DescribedClass
- RSpec/EmptyLineAfterSubject
- RSpec/NestedGroups
- RSpec/ExampleLength
- RSpec/MultipleExpectations